met-api/src/met_api/services/comment_service.py

```diff
@@ -79,7 +80,10 @@ def extract_comments(cls, survey_submission: SubmissionSchema, survey: SurveySch
     def review_comment(cls, comment_id, status_id, external_user_id):
         """Review comment."""
         user = UserService.get_user_by_external_id(external_user_id)
-        if not status_id or status_id == 1 or not user:
+
+        valid_statuses = [status.id for status in CommentStatus.get_comment_statuses()]
+
+        if not status_id or status_id == 1 or status_id not in valid_statuses or not user:
```

Comment:
please replace 1 with CommentStatus.Pending.Value that way we know whats the 1 for

Reply:
of course